doc/content/devices/arduino/_index.md

```diff
@@ -0,0 +1,24 @@
+---
+title: The Things Network Arduino API
+aliases:
+ - /refactor/arduino/
+ - /v2-preview/arduino/
+ - /current/arduino/
+ - /arduino/
+---
+
+[Arduino](https://www.arduino.cc/en/Guide/Introduction) is an open-source electronics platform based on easy-to-use hardware and software. It's intended for anyone making interactive projects.
+
+With [The Things Network Library](https://github.com/thethingsnetwork/arduino-device-lib) Arduino Boards can communicate via The Things Network. Both [The Things Uno]({{< relref "../the-things-uno" >}}) and [The Things Node]({{< relref "../the-things-node" >}}) are Arduino Boards that include a LoRaWAN module.
```

Comment:
Is this relevant anymore? Are those libraries updated for devices to use TTS? "communicate via The Things Network" doesn't seem right to me, are we talking about TTS CE?